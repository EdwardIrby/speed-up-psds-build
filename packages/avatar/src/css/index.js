import {
  type,
  colorsTextIcon,
  colorsBackgroundUtility
} from '@pluralsight/ps-design-system-core'

import * as vars from '../vars/index.js'

export default {
  '.psds-avatar': {
    overflow: 'hidden',
    display: 'inline-block',
    textAlign: 'center',
    position: 'relative',
    borderRadius: '100%'
  },

  [`.psds-avatar--size-${vars.sizes.xSmall}`]: {
    width: vars.widths.xSmall,
    height: vars.widths.xSmall,
    fontSize: type.fontSizeXSmall
  },
  [`.psds-avatar--size-${vars.sizes.small}`]: {
    width: vars.widths.small,
    height: vars.widths.small,
    fontSize: type.fontSizeSmall
  },
  [`.psds-avatar--size-${vars.sizes.medium}`]: {
    width: vars.widths.medium,
    height: vars.widths.medium,
    fontSize: type.fontSizeMedium
  },
  [`.psds-avatar--size-${vars.sizes.large}`]: {
    width: vars.widths.large,
    height: vars.widths.large,
    fontSize: type.fontSizeLarge
  },
  [`.psds-avatar--size-${vars.sizes.xLarge}`]: {
    width: vars.widths.xLarge,
    height: vars.widths.xLarge,
    fontSize: type.fontSizeXLarge
  },

  '.psds-avatar__image': {
    position: 'absolute',
    top: '0',
    left: '0',
    width: '100%',
    height: '100%',
    objectFit: 'cover'
  },

  '.psds-avatar__initials': {
    width: 'auto',
    height: '100%',
    display: 'flex',
    flexDirection: 'column',
    justifyContent: 'center',
    color: colorsTextIcon.highOnDark,
    fontWeight: type.fontWeightBook,
    backgroundColor: colorsBackgroundUtility.base
  }
}
