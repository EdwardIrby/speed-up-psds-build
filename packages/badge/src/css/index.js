import core from '@pluralsight/ps-design-system-core'

export default {
  '.psds-badge': {
    display: 'inline-block',
    padding: `0 ${core.layout.spacingXSmall}`,
    fontWeight: core.type.fontWeightMedium,
    fontSize: core.type.fontSizeXSmall,
    lineHeight: core.type.lineHeightStandard,
    borderRadius: '2px',
    textTransform: 'uppercase'
  },
  // --appearance-stroke
  '.psds-badge--appearance-stroke.psds-badge--color-gray': {
    color: core.colors.gray02,
    border: `1px solid ${core.colors.gray02}`
  },
  '.psds-badge--appearance-stroke.psds-badge--color-green': {
    color: core.colors.green,
    border: `1px solid ${core.colors.green}`
  },
  '.psds-badge--appearance-stroke.psds-badge--color-yellow': {
    color: core.colors.yellow,
    border: `1px solid ${core.colors.yellow}`
  },
  '.psds-badge--appearance-stroke.psds-badge--color-red': {
    color: core.colors.red,
    border: `1px solid ${core.colors.red}`
  },
  '.psds-badge--appearance-stroke.psds-badge--color-blue': {
    color: core.colors.blue,
    border: `1px solid ${core.colors.blue}`
  },
  // --appearance-default
  '.psds-badge--appearance-default.psds-badge--color-gray': {
    color: core.colors.white,
    backgroundColor: core.colors.gray03,
    border: `1px solid ${core.colors.gray03}`
  },
  '.psds-badge--appearance-default.psds-badge--color-green': {
    color: core.colors.white,
    backgroundColor: core.colors.green,
    border: `1px solid ${core.colors.green}`
  },
  '.psds-badge--appearance-default.psds-badge--color-yellow': {
    color: core.colors.black,
    backgroundColor: core.colors.yellow,
    border: `1px solid ${core.colors.yellow}`
  },
  '.psds-badge--appearance-default.psds-badge--color-red': {
    color: core.colors.white,
    backgroundColor: core.colors.red,
    border: `1px solid ${core.colors.red}`
  },
  '.psds-badge--appearance-default.psds-badge--color-blue': {
    color: core.colors.white,
    backgroundColor: core.colors.blue,
    border: `1px solid ${core.colors.blue}`
  }
}
