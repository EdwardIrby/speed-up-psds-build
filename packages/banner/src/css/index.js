import {
  colorsTextIcon,
  type,
  layout,
  motion,
  colorsBlue,
  colorsGreen,
  colorsYellow,
  colorsRed
} from '@pluralsight/ps-design-system-core'

import * as vars from '../vars/index.js'

export default {
  '.psds-banner': {
    display: 'flex',
    alignItems: 'flex-start',
    width: '100%',
    overflow: 'hidden',
    fontSize: type.fontSizeSmall,
    lineHeight: type.lineHeightStandard,
    fontWeight: type.fontWeightMedium,
    padding: `${layout.spacingXSmall} ${layout.spacingSmall}`
  },

  [`.psds-banner--color-${vars.colors.blue}`]: {
    color: colorsTextIcon.highOnDark,
    backgroundColor: colorsBlue.base
  },
  [`.psds-banner--color-${vars.colors.green}`]: {
    color: colorsTextIcon.highOnDark,
    backgroundColor: colorsGreen.base
  },
  [`.psds-banner--color-${vars.colors.yellow}`]: {
    color: colorsTextIcon.highOnLight,
    backgroundColor: colorsYellow.base
  },
  [`.psds-banner--color-${vars.colors.red}`]: {
    color: colorsTextIcon.highOnDark,
    backgroundColor: colorsRed.base
  },

  '.psds-banner__button': {
    '&:hover, &:active, &:focus': {
      transition: `all ${motion.speedNormal}`,
      opacity: '0.65'
    }
  },

  '.psds-banner__text': {
    flex: '1',
    textAlign: 'center',

    '& a': {
      color: 'inherit',
      cursor: 'pointer',
      textDecoration: 'underline',

      '&:hover, &:active, &:focus': {
        color: 'inherit',
        transition: `all ${motion.speedNormal}`,
        opacity: '0.65'
      }
    }
  },

  '.psds-banner__dismiss': {
    position: 'relative',
    display: 'flex',
    alignItems: 'flex-start',
    background: 'none',
    border: 'none',
    marginLeft: layout.spacingSmall,
    padding: '0',
    color: 'currentColor',
    cursor: 'pointer',

    '&:hover, &:focus, &:active': {
      outline: 'none',
      borderRadius: '2px',
      boxShadow: '0 0 2px currentColor, inset 0 0 2px currentColor'
    }
  }
}
