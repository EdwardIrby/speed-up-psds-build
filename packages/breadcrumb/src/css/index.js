import core from '@pluralsight/ps-design-system-core'

export default {
  '.psds-breadcrumb': {
    paddingTop: core.layout.spacingMedium,
    paddingBottom: core.layout.spacingMedium
  }
}
