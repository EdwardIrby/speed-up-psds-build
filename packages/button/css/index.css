@import "@pluralsight/ps-design-system-core";

.ps-button {
  display: inline-block;
  padding: 8px 16px;
  border: 0;
  border-radius: 2px;

  font-size: var(--psTypeFontSizeSmall);
  line-height: var(--psTypeLineHeightStandard);
  font-weight: var(--psTypeFontWeightMedium);
  text-align: center;

  /* TODO: font family */
  color: #fff;
  background: var(--psColorsOrange);
  cursor: pointer;
}

/* size */
.ps-button--tiny {
  font-size: var(--psTypeFontSizeExtraSmall);
  line-height: var(--psTypeLineHeightStandard);
}
.ps-button--small {
  font-size: var(--psTypeFontSizeSmall);
  line-height: var(--psTypeLineHeightStandard);
}
.ps-button--medium {
  font-size: var(--psTypeFontSizeSmall);
  line-height: var(--psTypeLineHeightStandard);
}
.ps-button--large {
  font-size: var(--psTypeFontSizeMedium);
  line-height: var(--psTypeLineHeightExtra);
}

/* appearance */
.ps-button--stroke {
  border: 1px solid var(--psColorsOrange);
  color: var(--psColorsOrange);
  background: none;
}
.ps-button--flat {
  border: none;
  color: var(--psColorsGray02);
  background: none;
}
.ps-button--flat:hover {
  color: var(--psColorsWhite);
  background: var(--psColorsGray04);
}

/* icon and iconAlign */

/* disabled */
.ps-button--disabled {
  color: var(--psColorsGray02);
}
.ps-button--disabled:not(.ps-button--flat) {
  background: var(--psColorsGray04);
}
.ps-button--disabled.ps-button--flat {
  opacity: 0.4;
}
.ps-button--disabled.ps-button--flat:hover {
  color: var(--psColorsGray02);
  background: none;
}