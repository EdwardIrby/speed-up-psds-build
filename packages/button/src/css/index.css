@keyframes psds-button__keyframes__spin {
     100% {
         transform: rotate(360deg);
    }
}
.psds-button__loading {
     display: inline-block;
     height: calc(100% - 4px);
     width: calc(100% - 4px);
     margin: 2px;
     border-style: solid;
     border-width: 2px;
     border-radius: 100%;
     animation: psds-button__keyframes__spin 1s linear infinite;
}
.psds-button{
    position:relative;
    display:-webkit-inline-box;
    display:-moz-inline-box;
    display:-ms-inline-flexbox;
    display:-webkit-inline-flex;
    display:inline-flex;
    justify-content:center;
    align-items:stretch;
    border:0;
    border-radius:3px;
    font-size:var(--psTypeFontSize300);
    font-weight:var(--psTypeFontWeight500);
    text-align:center;
    white-space:nowrap;
    text-decoration:none;
    transition:background var(--psMotionSpeedFast),-webkit-transform var(--psMotionSpeedXFast),transform var(--psMotionSpeedXFast);
    vertical-align:middle;
    -webkit-box-pack:center;
    -webkit-justify-content:center;
    -webkit-box-align:stretch;
    -webkit-align-items:stretch;
    -webkit-transition:background var(--psMotionSpeedFast),-webkit-transform var(--psMotionSpeedXFast),transform var(--psMotionSpeedXFast);
    -moz-transition:background var(--psMotionSpeedFast),transform var(--psMotionSpeedXFast)
}
.psds-button:not([disabled]):hover{
    cursor:pointer
}
.psds-button:focus:not(:focus-visible){
    outline:0;
    box-shadow:none
}
.psds-button:not([disabled]):active{
    transform:scale(.98);
    -webkit-transform:scale(.98)
}
.psds-button--dark:focus{
    outline:0;
    box-shadow:0 0 0 2px var(--psColorsBlue4)
}
.psds-button--dark:focus-visible{
    outline:0;
    box-shadow:0 0 0 2px var(--psColorsBlue4)
}
.psds-button--light:focus{
    outline:0;
    box-shadow:0 0 0 2px var(--psColorsBlue8)
}
.psds-button--light:focus-visible{
    outline:0;
    box-shadow:0 0 0 2px var(--psColorsBlue8)
}
.psds-button--layout-fullWidth{
    display:-webkit-box;
    display:-moz-box;
    display:-ms-flexbox;
    display:-webkit-flex;
    display:flex;
    width:100%
}
.psds-button--size-xSmall{
    font-size:var(--psTypeFontSize100);
    letter-spacing:var(--psTypeLetterSpacingLooser);
    padding:4px 8px;
    height:24px
}
.psds-button--size-small{
    font-size:var(--psTypeFontSize200);
    letter-spacing:var(--psTypeLetterSpacingLooser);
    padding:6px 12px;
    height:32px
}
.psds-button--size-medium{
    font-size:var(--psTypeFontSize300);
    letter-spacing:var(--psTypeLetterSpacingLoose);
    padding:10px 16px;
    height:40px
}
.psds-button--size-large{
    font-size:var(--psTypeFontSize400);
    letter-spacing:var(--psTypeLetterSpacingNone);
    padding:12px 20px;
    height:48px
}
.psds-button--appearance-primary-theme-dark{
    color:var(--psColorsTextIconHighOnDark);
    background:var(--psColorsPrimaryActionBackground)
}
.psds-button--appearance-primary-theme-dark:not([disabled]):hover{
    background:var(--psColorsBlue8);
    cursor:pointer
}
.psds-button--appearance-primary-theme-dark:not([disabled]):focus{
    background:var(--psColorsBlue8);
    cursor:pointer
}
.psds-button--appearance-primary-theme-dark:not([disabled]):active{
    background:var(--psColorsBlue9)
}
.psds-button--appearance-primary-theme-light{
    color:var(--psColorsTextIconHighOnDark);
    background:var(--psColorsPrimaryActionBackground)
}
.psds-button--appearance-primary-theme-light:not([disabled]):hover{
    background:var(--psColorsBlue8);
    cursor:pointer
}
.psds-button--appearance-primary-theme-light:not([disabled]):focus{
    background:var(--psColorsBlue8);
    cursor:pointer;
    box-shadow:0 0 0 2px var(--psColorsBlue5)
}
.psds-button--appearance-primary-theme-light:not([disabled]):focus-visible{
    box-shadow:0 0 0 2px var(--psColorsBlue5)
}
.psds-button--appearance-primary-theme-light:not([disabled]):active{
    background:var(--psColorsBlue9)
}
.psds-button--appearance-secondary-theme-dark{
    color:var(--psColorsTextIconHighOnDark);
    background:var(--psColorsBackgroundUtility25)
}
.psds-button--appearance-secondary-theme-dark:not([disabled]):hover{
    background:var(--psColorsBackgroundUtility30)
}
.psds-button--appearance-secondary-theme-dark:not([disabled]):focus{
    background:var(--psColorsBackgroundUtility30)
}
.psds-button--appearance-secondary-theme-dark:not([disabled]):active{
    background:var(--psColorsBackgroundUtility20)
}
.psds-button--appearance-secondary-theme-light{
    color:var(--psColorsTextIconHighOnLight);
    background:var(--psColorsBackgroundUtility20)
}
.psds-button--appearance-secondary-theme-light:not([disabled]):hover{
    background:var(--psColorsBackgroundUtility25)
}
.psds-button--appearance-secondary-theme-light:not([disabled]):focus{
    background:var(--psColorsBackgroundUtility25)
}
.psds-button--appearance-secondary-theme-light:not([disabled]):active{
    background:var(--psColorsBackgroundUtility30)
}
.psds-button--appearance-stroke-theme-dark{
    border:2px solid var(--psColorsPrimaryActionBackground);
    color:var(--psColorsPrimaryActionTextDarkTheme);
    background:0 0
}
.psds-button--appearance-stroke-theme-dark:not([disabled]):hover{
    border:2px solid var(--psColorsBlue5);
    background:rgba(10,NaN,NaN,.15);
    color:var(--psColorsBlue4)
}
.psds-button--appearance-stroke-theme-dark:not([disabled]):focus{
    border:2px solid var(--psColorsBlue5);
    background:rgba(10,NaN,NaN,.15);
    color:var(--psColorsBlue4)
}
.psds-button--appearance-stroke-theme-dark:not([disabled]):active{
    border:2px solid var(--psColorsBlue7);
    background:rgba(10,NaN,NaN,.2)
}
.psds-button--appearance-stroke-theme-light{
    border:2px solid var(--psColorsPrimaryActionBackground);
    color:var(--psColorsPrimaryActionTextLightTheme);
    background:0 0
}
.psds-button--appearance-stroke-theme-light:not([disabled]):hover{
    border:2px solid var(--psColorsBlue7);
    background:rgba(10,NaN,NaN,.05);
    color:var(--psColorsBlue8)
}
.psds-button--appearance-stroke-theme-light:not([disabled]):focus{
    border:2px solid var(--psColorsBlue7);
    background:rgba(10,NaN,NaN,.05);
    color:var(--psColorsBlue8)
}
.psds-button--appearance-stroke-theme-light:not([disabled]):active{
    border:2px solid var(--psColorsBlue8);
    background:rgba(10,NaN,NaN,.1)
}
.psds-button--appearance-flat{
    border:none;
    background:0 0
}
.psds-button--appearance-flat-theme-dark{
    color:var(--psColorsTextIconLowOnDark)
}
.psds-button--appearance-flat-theme-dark:not([disabled]):hover{
    background:var(--psColorsBackgroundUtility25);
    color:var(--psColorsTextIconHighOnDark)
}
.psds-button--appearance-flat-theme-dark:not([disabled]):focus{
    background:var(--psColorsBackgroundUtility25);
    color:var(--psColorsTextIconHighOnDark)
}
.psds-button--appearance-flat-theme-dark:not([disabled]):active{
    background:var(--psColorsBackgroundUtility20);
    color:var(--psColorsTextIconHighOnDark)
}
.psds-button--appearance-flat-theme-light{
    color:var(--psColorsTextIconLowOnLight)
}
.psds-button--appearance-flat-theme-light:not([disabled]):hover{
    background:var(--psColorsBackgroundUtility20);
    color:var(--psColorsTextIconHighOnLight)
}
.psds-button--appearance-flat-theme-light:not([disabled]):focus{
    background:var(--psColorsBackgroundUtility20);
    color:var(--psColorsTextIconHighOnLight)
}
.psds-button--appearance-flat-theme-light:not([disabled]):active{
    background:var(--psColorsBackgroundUtility15);
    color:var(--psColorsTextIconHighOnLight)
}
.psds-button--disabled{
    opacity:50%;
    cursor:not-allowed
}
.psds-button--disabled:not([disabled]):hover{
    color:inherit;
    background:inherit;
    border:inherit
}
.psds-button--disabled{
    opacity:.5
}
.psds-button--iconAlign-right{
    flex-direction:row-reverse;
    -webkit-box-orient:horizontal;
    -webkit-box-direction:reverse;
    -webkit-flex-direction:row-reverse
}
.psds-button--iconAlign-right-size-xSmall{
    padding-right:4px
}
.psds-button--iconAlign-left-size-xSmall{
    padding-left:4px
}
.psds-button--iconAlign-right-size-small{
    padding-right:8px
}
.psds-button--iconAlign-left-size-small{
    padding-left:8px
}
.psds-button--iconAlign-right-size-medium{
    padding-right:12px
}
.psds-button--iconAlign-left-size-medium{
    padding-left:12px
}
.psds-button--iconAlign-right-size-large{
    padding-right:16px
}
.psds-button--iconAlign-left-size-large{
    padding-left:16px
}
.psds-button--iconOnly{
    padding:0
}
.psds-button--iconOnly-size-xSmall{
    width:24px
}
.psds-button--iconOnly-size-small{
    width:32px
}
.psds-button--iconOnly-size-medium{
    width:40px
}
.psds-button--iconOnly-size-large{
    width:48px
}
.psds-button__icon{
    display:-webkit-box;
    display:-moz-box;
    display:-ms-flexbox;
    display:-webkit-flex;
    display:flex;
    align-items:center;
    min-height:100%;
    -webkit-box-align:center;
    -webkit-align-items:center
}
.psds-button__icon--iconAlign-right{
    margin-left:8px;
    margin-right:0
}
.psds-button__icon--iconAlign-left{
    margin-left:0;
    margin-right:8px
}
.psds-button__icon--iconAlign-right-size-xSmall{
    margin-left:4px;
    margin-right:0
}
.psds-button__icon--iconAlign-left-size-xSmall{
    margin-left:0;
    margin-right:4px
}
.psds-button__icon--iconAlign-right-size-small{
    margin-left:4px;
    margin-right:0
}
.psds-button__icon--iconAlign-left-size-small{
    margin-left:0;
    margin-right:4px
}
.psds-button__icon--iconOnly{
    justify-content:center;
    width:100%;
    margin:0;
    -webkit-box-pack:center;
    -webkit-justify-content:center
}
.psds-button__loading--appearance-primary{
    border-color:var(--psColorsBorderLowOnDark);
    border-top-color:var(--psColorsTextIconHighOnDark)
}
.psds-button__loading--appearance-secondary-theme-light{
    border-color:var(--psColorsBorderLowOnLight);
    border-top-color:var(--psColorsTextIconHighOnLight)
}
.psds-button__loading--appearance-secondary-theme-dark{
    border-color:var(--psColorsBorderLowOnDark);
    border-top-color:var(--psColorsTextIconHighOnDark)
}
.psds-button__loading--appearance-stroke-theme-light{
    border-color:var(--psColorsBorderLowOnLight);
    border-top-color:var(--psColorsPrimaryActionBackground)
}
.psds-button__loading--appearance-stroke-theme-dark{
    border-color:var(--psColorsBorderLowOnDark);
    border-top-color:var(--psColorsPrimaryActionBackground)
}
.psds-button__loading--appearance-flat-theme-light{
    border-color:var(--psColorsBorderLowOnLight);
    border-top-color:var(--psColorsTextIconHighOnLight)
}
.psds-button__loading--appearance-flat-theme-dark{
    border-color:var(--psColorsBorderLowOnDark);
    border-top-color:var(--psColorsTextIconHighOnDark)
}
.psds-button__text{
    align-items:center;
    display:-webkit-inline-box;
    display:-moz-inline-box;
    display:-ms-inline-flexbox;
    display:-webkit-inline-flex;
    display:inline-flex;
    pointer-events:none;
    -webkit-box-align:center;
    -webkit-align-items:center
}
