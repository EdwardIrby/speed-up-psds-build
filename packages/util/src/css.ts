type PrimitiveValueOptions = {
  [key: string]: boolean
}
const primitiveValues: PrimitiveValueOptions = {
  string: true,
  boolean: true,
  number: true
}
const trueTypeOf = (obj?: unknown) =>
  Object.prototype.toString.call(obj).slice(8, -1).toLowerCase()

type CssArgs = Array<number | string | boolean | undefined | null | void>

const reduceWhitespace = (str: string) => str.replace(/^(\s\s+|\n)/gm, '')
const isPrimitive = (val: unknown) => primitiveValues[trueTypeOf(val)] || false
export const css = (
  literalSections: TemplateStringsArray,
  ...substs: CssArgs
) => {
  const { raw } = literalSections
  const result = substs.reduce<CssArgs>((acc, cur, i) => {
    acc.push(reduceWhitespace(raw[i]))
    const string = Array.isArray(cur)
      ? cur.filter(isPrimitive).join('')
      : isPrimitive(cur)
      ? cur
      : ''
    acc.push(string)
    return acc
  }, [])
  result.push(reduceWhitespace(raw[raw.length - 1]))
  return result.filter(Boolean).join('').trim()
}
