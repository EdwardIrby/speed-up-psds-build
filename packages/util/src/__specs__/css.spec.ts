import { css } from '../css'

test('css()', () => {
  const set = css`
    :host(:not(:last-child)) {
      display: none;
    }
    .ExampleOuter {
      grid-template-columns: 1fr 60px;
    }
  `
  const returnUndefined = (): undefined => undefined
  const returnVoid = () => {}
  const returnNull = () => null
  const actual = css`
    ${set}
    .ExampleOuter {
      color: white;
    }
    ${returnUndefined()}
    ${returnVoid()}
    ${returnNull()}
    .ExampleInner {
      background-color: blue;
      padding: 0;
    }
    [data-active='${true}'] {
      color: red;
      margin-left: auto;
    }
  `
  console.log(actual)
  expect(actual).toMatchSnapshot()
})
