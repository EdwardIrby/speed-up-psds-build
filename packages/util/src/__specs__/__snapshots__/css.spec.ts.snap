// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`css() 1`] = `
":host(:not(:last-child)) {
display: none;
}
.ExampleOuter {
grid-template-columns: 1fr 60px;
}.ExampleOuter {
color: white;
}
.ExampleInner {
background-color: blue;
padding: 0;
}
[data-active='true'] {
color: red;
margin-left: auto;
}"
`;
